tests/loggers/test_mlflow.py

```diff
@@ -1,9 +1,36 @@
+import os
+
+from pytorch_lightning import Trainer
 from pytorch_lightning.loggers import MLFlowLogger
+from tests.base import EvalModelTemplate
 
 
 def test_mlflow_logger_exists(tmpdir):
-    """Verify that basic functionality of mlflow logger works."""
+    """ Test launching two independent loggers. """
     logger = MLFlowLogger('test', save_dir=tmpdir)
-    # Test already exists
+    # same name leads to same experiment id, but different runs get recorded
     logger2 = MLFlowLogger('test', save_dir=tmpdir)
+    assert logger.experiment_id == logger2.experiment_id
     assert logger.run_id != logger2.run_id
+    logger3 = MLFlowLogger('new', save_dir=tmpdir)
+    assert logger3.experiment_id != logger.experiment_id
+
+
+def test_mlflow_logger_dirs_creation(tmpdir):
```

Comment:
I'm worried that this will affect the other loggers...
Can you also add the same test for the other loggers?